mne_bids/config.py

```diff
@@ -317,13 +343,26 @@
     + coordsys_wildcard
 )
 
+
+# EMG allows arbitrary (user-defined) coord frames
+class EmgCoordFrames(list):
+    """Container for arbitrary (user-defined) coordinate frames (spaces)."""
+
+    def __contains__(self, item):
+        """Pretends to contain any non-empty string."""
+        return isinstance(item, str) and len(item)
+
+
+BIDS_EMG_COORDINATE_FRAMES = EmgCoordFrames()
```

Comment:
this may be a bit too clever; EMG-BIDS allows coordsys spaces to have arbitrary (user-defined) names, and this is a way to make that possible (avoid "space entity is not valid for datatype EMG" errors) without disabling the existing checking mechanisms.  Open to other approaches if this doesn't sit well with somebody.